blackmarket/Core/Extensions/Color.swift

```diff
@@ -0,0 +1,44 @@
+//
+//  Extension+Color.swift
+//  blackmarket
+//
+//  Created by Daniel Parra on 9/28/22.
+//
+
+import SwiftUI
+
+extension Color {
+    static let ui = Color.UI()
+    
+    struct UI {
```

Comment:
```suggestion
    enum UI {
```

we don't need to instantiate the UI structure since it is merely an enclosing namespace.